src/main/resources/liquibase/changelog.xml

```diff
@@ -115,6 +115,7 @@
     <include file="liquibase/202307171013-add-indexes-for-tally-instance-view-queries.xml"/>
     <include file="/liquibase/202307131456-remove-granularity-from-billable-usage-remittance.xml"/>
     <include file="liquibase/202307051132-rename-instance_id-to-host_id.xml"/>
+    <include file="liquibase/202307261851-event-change-event-type-data.xml"/>
```

Comment:
Please move this line down below the commented out changeset so the comment over it makes sense.